web/src/features/AppConfig/components/AppConfig.tsx

```diff
@@ -629,7 +629,9 @@ class AppConfig extends Component<Props, State> {
                   </div>
                   {group.items ? (
                     <div className="side-nav-items">
-                      {group.items?.map((item, j) => {
+                      {group.items
+                      ?.filter((item) => item.type !== "label")
```

Comment:
simple fix - just filter out the label types before mapping the config components